TailLight/TailLight.vcxproj

```diff
@@ -87,6 +90,9 @@ copy ..\IntelliMouse.ps1 $(OutDir)</Command>
       <PreprocessorDefinitions>%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ExceptionHandling>
       </ExceptionHandling>
+      <ProgramDataBaseFileName>$(OutDir)$(TargetName).pdb</ProgramDataBaseFileName>
+      <AssemblerOutput>All</AssemblerOutput>
+      <PreprocessToFile>false</PreprocessToFile>
```

Comment:
What are you trying to achieve with these project settings? PDB files are already being generated for both debug & release builds.